### ISSUE TITLE
Typescript 'Image' missing 'source' property

### ISSUE THREAD

**Issue by RoniFinTech** [GeekyAnts/NativeBase#2908]
```
<Image
source={{ uri: d.picUrl[0] }}
style={{ height: 200, width: 200, flex: 1 }}
/>
```

This is the error message:

> Type '{ source: { uri: string; }; style: { height: number; width: number; flex: number; }; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Image> & Readonly<Image> & Readonly<{ children?: ReactNode; }>'.
>   Property 'source' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Image> & Readonly<Image> & Readonly<{ children?: ReactNode; }>'.ts(2322)

**Comment by sankhadeeproy007** [comment]
I just pushed a fix for this in master? Can you point your package.json to master and check if it works?

**Comment by sankhadeeproy007** [comment]
Fixed in 2.13.8
